reactive/media/jackson/src/main/java/module-info.java

```diff
@@ -23,7 +25,12 @@
  *
  * @see JacksonSupport
  */
+@Feature(value = "Jackson",
+        description = "Media support for Jackson",
+        path = {"WebServer", "WebClient", "Jackson"},
```

Comment:
This will not work - this will create path `WebServer/WebClient/Jackson`
Please change to root node ("Jackson")